src/task_queue/nudges/user_event_nudge.py

```diff
@@ -134,52 +163,67 @@ def get_logo(event):
 def convert_date(date, format):
     return date.astimezone(eastern_tz).strftime(format)
 
+
 def get_date_range(start, end):
-    start_date =start.strftime('%b-%d-%Y')
-    end_date = end.strftime('%b-%d-%Y')
+    start_date = start.strftime("%b-%d-%Y")
+    end_date = end.strftime("%b-%d-%Y")
     if start_date == end_date:
-        return f"{convert_date(start,'%b %d, %Y')}, {convert_date(start,' %I:%M %p')} - {convert_date(end,' %I:%M %p')}"
-    return f"{convert_date(start,'%b %d, %Y %I:%M %p')} - {convert_date(end,'%b %d, %Y %I:%M %p')}"
+        return f"{convert_date(start,'%b %d, %Y')}, {convert_date(start,' %H:%M %p')} - {convert_date(end,' %H:%M %p')}"
```

Comment:
I had just made a change to %I to show the dates in 12 hour format.  Did you change it back on purpose to %H?

Reply:
In that case, can you test that it gives the time in 12 hour format?  That is what users want.